From d4cb07892143a2ae1f452b46bf5111a40ef7f466 Mon Sep 17 00:00:00 2001
From: Will Kahn-Greene <will.guaraldi@pculture.org>
Date: Tue, 21 Jun 2011 14:59:57 -0400
Subject: [PATCH] 17817. Changes timeout from 4 to 6 seconds

We're theorizing that on the linux build box a 4 second timeout isn't
long enough because it's slow and old.
---
 tv/lib/test/subprocesstest.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tv/lib/test/subprocesstest.py b/tv/lib/test/subprocesstest.py
index 529a7c1..ac7546b 100644
--- a/tv/lib/test/subprocesstest.py
+++ b/tv/lib/test/subprocesstest.py
@@ -91,7 +91,7 @@ class SubprocessManagerTest(EventLoopTest):
         self.subprocess.shutdown()
         EventLoopTest.tearDown(self)
 
-    def _wait_for_subprocess_ready(self, timeout=4.0):
+    def _wait_for_subprocess_ready(self, timeout=6.0):
         """Wait for the subprocess to startup."""
 
         start = time.time()
-- 
1.7.6

